HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -91,6 +90,27 @@ string custodianCode
             };
             SendEmail(emailModel);
         }
+        public void SendReferenceCodePendingLAEmail
+        (
+            string emailAddress,
+            string recipientName,
+            string referenceCode,
+            string custodianCode
+        )
+        {
+            SendReferenceCodeEmail(emailAddress, recipientName, referenceCode, custodianCode, govUkNotifyConfig.ReferenceCodePendingLATemplate);
+        }
+
+        public void SendReferenceCodeLiveLAEmail
```

Comment:
Let's change both of these public methods and the private one to take a `ReferralRequest` object, from which it can pick out the `referenceCode` and `custodianCode`.